lib/decisionmaking-flow.js

```diff
@@ -199,7 +199,12 @@ INSERT DATA {
       dct:created ${sparqlEscapeDateTime(now)} ;
       ext:modified ${sparqlEscapeDateTime(now)} ;
       dct:type ${sparqlEscapeUri(subcaseType)} ;
-      ext:agendapuntType ${sparqlEscapeUri(agendaItemType)} .
+      ext:agendapuntType ${sparqlEscapeUri(agendaItemType)} ;
+      ext:bekrachtigdDoor ?mandatee .
+  }
+} WHERE {
+  GRAPH ${sparqlEscapeUri(KANSELARIJ_GRAPH_URI)} {
+    ${sparqlEscapeUri(latestSubcaseUri)} ext:heeftBevoegde ?mandatee .
```

Comment:
instead of passing in mandatees like the other PR.
We could just make sure the `where` portion always returns something and make mandatees optional?
That way, we can also add the governmentareas in the where part here.

we could make the where `${sparqlEscapeUri(decisionmakingFlowUri)} a ?decisionMakingFlow` ?

Reply:
> Tested without any mandatees. No insert happened Assuming having no previous subcase is not really possible when we know the pobj and the decisionmaking flow id
> 
> ![image](https://private-user-images.githubusercontent.com/22245223/325909007-0b79a060-7405-4799-9a19-0da4ffc9b53a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQxMjQ3ODksIm5iZiI6MTcxNDEyNDQ4OSwicGF0aCI6Ii8yMjI0NTIyMy8zMjU5MDkwMDctMGI3OWEwNjAtNzQwNS00Nzk5LTlhMTktMGRhNGZmYzliNTNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI2VDA5NDEyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ5NTJlOGU1ZTE2ZmVlNzlmMWNjMzkwOWUwMGM1MWFlODAzYjNlNDVhOWFjNjI4NjQ1NzFiZDAxZGZhZWFmODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ax3CmjL78lQqswR_9l4et1gEXuIqHlEXLHuDyYylGQY)

I cannot reproduce this.

Submitted subcase without mandatees.
![image](https://github.com/kanselarij-vlaanderen/vlaams-parlement-sync-service/assets/22411874/855765f4-a497-4e41-82d5-4a14974a4941)

Generated subcase of bekrachtiging which exists and correctly has no mandatees for bekrachtiging.
![image](https://github.com/kanselarij-vlaanderen/vlaams-parlement-sync-service/assets/22411874/fc59c50d-b806-4b68-ab8d-fd03ee4c29f4)

Your screenshot has `Insert into <(NULL)>`, did you correctly replace all graphs before trying it out against the SPARQL endpoint?

Reply:
> instead of passing in mandatees like the other PR passes in government areas We could just make sure the `where` portion always returns something and make mandatees optional? That way, we can also add the governmentareas in the where part here.
> 
> we could make the where something like
> 
> ```
> ${sparqlEscapeUri(decisionmakingFlowUri)} a ?decisionMakingFlow
> ${sparqlEscapeUri(decisionmakingFlowUri)} dossier:doorloopt ${sparqlEscapeUri(subcaseUri)} .
> OPTIONAL ?mandatees
> OPTIONAL ?governmentAreas
> ```

Whatever you prefer/is the least amount of work to change, I don't have a preference one way or another.

Reply:
I grabbed the Query from the chrome inspector.
So should be ok.
After I added mandatees, I retried the query and it did insert 9 triples :man_shrugging:

Reply:
```
PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
PREFIX dct: <http://purl.org/dc/terms/>
PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
PREFIX besluitvorming: <https://data.vlaanderen.be/ns/besluitvorming#>

INSERT {
  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
    <http://themis.vlaanderen.be/id/besluitvormingsaangelegenheid/6374F30CD9A98BD0A2288557> dossier:doorloopt <http://themis.vlaanderen.be/id/procedurestap/b033ab60-03a8-11ef-a576-bf4eee6a1da6> .
    <http://themis.vlaanderen.be/id/procedurestap/b033ab60-03a8-11ef-a576-bf4eee6a1da6> a dossier:Procedurestap ;
      mu:uuid """b033ab60-03a8-11ef-a576-bf4eee6a1da6""" ;
      dct:alternative """over open scholen""" ;
      
      dct:created "2024-04-26T08:40:51.734Z"^^xsd:dateTime ;
      ext:modified "2024-04-26T08:40:51.734Z"^^xsd:dateTime ;
      dct:type <http://themis.vlaanderen.be/id/concept/procedurestap-type/bdba2bbc-7af6-490b-98a8-433955cfe869> ;
      ext:agendapuntType <http://themis.vlaanderen.be/id/concept/agendapunt-type/dd47a8f8-3ad2-4d5a-8318-66fc02fe80fd> ;
      ext:bekrachtigdDoor ?mandatee .
  }
} WHERE {
  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
    <http://themis.vlaanderen.be/id/procedurestap/6374F312D9A98BD0A2288558> ext:heeftBevoegde ?mandatee .
  }
}
```

This decisionmakingFlow is in the testdata, not sure what I am missing atm